.github/workflows/pr.yml

```diff
@@ -124,6 +126,14 @@ jobs:
   e2e-windows-subset:
     runs-on: windows-latest
     steps:
+      # Workaround for: https://github.com/bazel-contrib/bazel-lib/issues/968.
```

Comment:
Is this something we would like to have "permanently" in this config?

Reply:
There should be a TODO to remove if you're hinting at this 😄